schema/pet-insurance.json

```diff
@@ -40,101 +44,146 @@
             "inputMethod": "SelectOne"
         }
     },
     "outputs": {
+        "estimatedPrice": {
+            "description": "Emitted before asking for payment information.<br/>This is usually not the final amount to be paid, especially on websites which charge credit and company cards additionally. You will get <code>finalPrice</code> later.",
+            "typeRef": "#/domains/Generic/types/PriceConsent"
+        },
         "availableBreedTypes": {
-            "typeRef": "#/domains/PetInsurance/types/BreedTypes"
+            "typeRef": "#/domains/PetInsurance/types/BreedTypes",
+            "description": "Available breed typologies (Pedigree, Non-Pedigree etc)."
         },
         "finalPrice": {
             "description": "Emitted immediately before placing order, when final price is available.<br/>Automation will request <code>finalPriceConsent</code> input which should match this object.",
             "typeRef": "#/domains/Generic/types/PriceConsent"
         },
         "availableMaritalStatusOptions": {
+            "description": "Owner's marital status available options.",
             "typeRef": "#/domains/PetInsurance/types/MaritalStatusOptions"
         },
         "availableVoluntaryExcesses": {
+            "description": "Available voluntary excesses (in % increments).",
             "typeRef": "#/domains/PetInsurance/types/VoluntaryExcesses"
         },
         "availableAddresses": {
+            "description": "Available addresses after postcode input.",
             "typeRef": "#/domains/PetInsurance/types/Addresses"
         },
         "availableCovers": {
+            "description": "Available protection covers.",
             "typeRef": "#/domains/PetInsurance/types/Covers"
         },
         "availableCoverTypes": {
+            "description": "Available types of coverage.",
             "typeRef": "#/domains/PetInsurance/types/CoverTypes"
         },
-        "availableVetFeesLimit": {
-            "typeRef": "#/domains/PetInsurance/types/VetFeesLimit"
+        "availableVetFees": {
+            "description": "Availebel vet fee amounts covered.",
+            "typeRef": "#/domains/PetInsurance/types/VetFees"
         },
         "availableCoverOptions": {
+            "description": "Available optional extra packages.",
             "typeRef": "#/domains/PetInsurance/types/CoverOptions"
         },
         "availableVetPaymentTerms": {
-            "typeRef": "#/domains/PetInsurance/types/VetPaymentTerms"
+            "description": "Available types of vet bill coverage.",
+            "typeRef": "#/domains/PetInsurance/types/PaymentTerms"
         },
         "availablePaymentTerms": {
+            "description": "Available methods of payment (card, directdebit etc).",
             "typeRef": "#/domains/PetInsurance/types/PaymentTerms"
         },
+        "insuranceProductInformationDocument": {
+            "typeRef": "#/domains/PetInsurance/types/StructuredText",

```

Comment:
DRY (Don't Repeat Yourself). The `StructuredText` definition already exists in `Generic`. Use that instead of copy/pasting it in here, too, since it's identical.